composer.json

```diff
@@ -23,7 +23,7 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": "^7.0",
```

Comment:
We could also keep `>=` but `^` seems to be safer as we don't know what PHP 8 is going to bring.